astroid/helpers.py

```diff
@@ -129,7 +129,9 @@ def _object_type_is_subclass(
     # issubclass(object, (1, type)) raises TypeError
     for klass in class_seq:
         if isinstance(klass, util.UninferableBase):
-            raise AstroidTypeError("arg 2 must be a type or tuple of types")
+            raise AstroidTypeError(
```

Comment:
The use of `klass` here seems unnecessary as we always know the exact type (Uninferable). We probably just want to reference that?